scripts/start_all.sh

```diff
@@ -7,9 +7,15 @@ cdir=`cd $(dirname $0); pwd`
     ./download_asg_dependencies.sh
 )
 
+(
+    cd $cdir/../dependiences
+    ./download_do.sh
```

Comment:
download_do.sh is under `scripts`, not `dependencies`.

Reply:
made the change accordingly.